c2qa/__init__.py

```diff
@@ -5,4 +5,4 @@
 from c2qa.qumoderegister import QumodeRegister
 
 import c2qa.util
-import c2qa.kraus
+#import c2qa.kraus
```

Comment:
Oops, I think I commented this out a while ago and forgot about it. This is fixed by updating the required version of Qiskit.